### PR TITLE
Move UserView to Infrastructure layer

### LINKED ISSUE #22: Wrong namespace for App\Domain\User\Query ?!
Hi @jorge07 ,

First of all many thanks for taking the time to demonstrate a DDD-es-cqrs example with sf4. I use it to start up a new project, and doing ddd for some time but not sf4, it does help a lot.

However, I'm a bit sceptical about the location of the classes in App\Domain\User\Query ... Imho they should not be in the Domain namespace. Generally what we will get for the "Query" part are projections and only data necessary for views, right?!

I guess the Read repository interface is acceptable to keep in the Domain layer, but the UserView class seems not right to me ...

What's your thought on this?

Thanks

### PR DESCRIPTION
**Pull request by Lutacon**
Fixes #22

I was having problems with the previous merge 😢 

So this is the final PR.

### CODE REVIEW

File: config/packages/doctrine.yaml [hunk 1]
```diff
@@ -24,9 +24,9 @@ doctrine:
...
         mappings:
-            Domain:
+            Projections:
                 is_bundle: false
                 type: yml
-                dir: '%kernel.project_dir%/config/mapping/orm/query'
-                prefix: 'App\Domain'
-                alias: Domain
+                dir: '%kernel.project_dir%/config/mapping/orm/projections'
+                prefix: 'App'
```
Comment (jorge07) [inline config/packages/doctrine.yaml]: `doctrine:schema:validate` is failing due to mapping issues.

I think the best way to fix it is:
```
            Domain:
                is_bundle: false
                type: yml
                dir: '%kernel.project_dir%/config/mapping/orm/domain'
                prefix: 'App\Domain'
                alias: Domain
            Projections:
                is_bundle: false
                type: yml
                dir: '%kernel.project_dir%/config/mapping/orm/projections'
                prefix: 'App\Infrastructure'

```

and structure the mapping like:
```
orm/
  domain/
    User.ValueObject.Auth.Credentials.orm.yml
  projections/
    User.Query.UserView.orm.yml
```

**Comment by jorge07** [comment]
I'll add a schema validate to the ci test suites

File: config/packages/doctrine.yaml (hunk 1 above)
Comment (Lutacon) [reply config/packages/doctrine.yaml]: Ok, I will do that.

**Comment by Lutacon** [comment]
This time i will do a proper rebase 😃

**Comment by jorge07** [comment]
Thanks for the contribution!

**Pull request merged**